plotly.agent.nut

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2015 Electric Imp
+// This file is licensed under the MIT License
+// http://opensource.org/licenses/MIT
+
+class Plotly {
+    
+    // Use this function to get a timestamp that Plotly will automatically 
+    // recognize and style correctly.
+    static function getPlotlyTimestamp() {
```

Comment:
getPlotlyTimestamp should take an optional timestamp as a parameter. A great usecase for needing this is when developers collect a bunch of readings from the device (in offline mode) then send them as a group up to the agent.